.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go: ["1.15", "1.16"]
+        go: ["1.17", "1.18"]
```

Comment:
Feel free to add the newer versions of Go, but let's keep the old versions too. We need to ensure that heketi can be compiled on older OS platforms.

Reply:
I doubt this is going to compile with `1.15` or `1.16` versions, but let me add those and check as well.

Reply:
``` go build  -ldflags "-X main.HEKETI_VERSION=v10.0.0-87-gabe8b550-HEAD -extldflags '-z relro -z now'" -o heketi
Error: vendor/k8s.io/client-go/plugin/pkg/client/auth/exec/metrics.go:21:2: cannot find package "." in:
	/home/runner/go/src/github.com/heketi/heketi/vendor/io/fs```